xrdp/xrdp_encoder.c

```diff
@@ -132,6 +132,9 @@ xrdp_encoder_create(struct xrdp_mm *mm)
     g_snprintf(buf, 1024, "xrdp_%8.8x_encoder_term", pid);
     self->xrdp_encoder_term = g_create_wait_obj(buf);
     self->max_compressed_bytes = client_info->max_fastpath_frag_bytes & ~15;
+    self->frames_in_flight = client_info->max_unacknowledged_frame_count;
```

Comment:
no, max_unacknowledged_frame_count is the value received from the client.  frames_in_flight is our value, possibly modified for usage.